### PR TITLE
Add Aion platform

### PR DESCRIPTION
**Pull request by kolya182**
Fixes https://github.com/TrustWallet/blockatlas/issues/32 https://github.com/TrustWallet/private-trust-api/issues/266

### CODE REVIEW

File: coin/slip44.go [hunk 1]
```diff
@@ -8,14 +8,16 @@ type Coin struct {
 	Decimals uint   `json:"decimals"`
 }
 
+// List of coin indexes
```
Comment (vikmeup) [inline coin/slip44.go]: no need comment

File: go.mod [hunk 2]
```diff
@@ -3,6 +3,7 @@ module github.com/trustwallet/blockatlas
 go 1.12
 
 require (
+	github.com/davecgh/go-spew v1.1.1
```
Comment (vikmeup) [inline go.mod]: `go-spew` not used?

File: platform/aion/api.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	})
+	router.GET("/:address", getTransactions)
+}
+
+func getTransactions(c *gin.Context) {
+	trxs, err := client.GetTxsOfAddress(c.Param("address"))
+
+	if err != nil {
+		logrus.WithError(err).Errorf("Aion: Failed to get transactions for address %v", c.Param("address"))
+	}
```
Comment (vikmeup) [inline platform/aion/api.go]: add integration tests

File: go.mod (hunk 2 above)
Comment (terorie) [reply go.mod]: It's used in the tests. I don't like `go.mod` / `go.sum` it pollutes the commits. Maybe we should remove it.
@kolya182 You can use `export GO111MODULE=off` to disable go.mod if you want

File: coin/slip44.go (hunk 1 above)
Comment (kolya182) [reply coin/slip44.go]: Actually `go` recommends adding comments` https://blog.golang.org/godoc-documenting-go-code, plus linter was demanding adding it.

Comment (kolya182) [reply coin/slip44.go]: many open-source `go` projects do comments https://github.com/cosmos/cosmos-sdk/blob/develop/server/init.go https://github.com/trezor/blockbook/blob/master/db/sync.go

Comment (terorie) [reply coin/slip44.go]: Projects should follow godoc standard godoc.org/github.com/trustwallet/blockatlas
That’s why the linter was asking for it.
Feel free to add it but I’m going to document the whole source anyways once the code structure is set in stone.

File: loaders.go [hunk 4]
```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/trustwallet/blockatlas/platform/ripple"
 	"github.com/trustwallet/blockatlas/platform/stellar"
 	"github.com/trustwallet/blockatlas/platform/tezos"
+	"github.com/trustwallet/blockatlas/platform/aion"
```
Comment (terorie) [inline loaders.go]: Please use alphabetic sort for import (lint)

File: platform/aion/source/model.go [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package source
+
+type AionTransactionRPCResponse struct {
+	Content []AionTransaction 
+} 
+
+type AionTransaction struct {
```
Comment (terorie) [inline platform/aion/source/model.go]: No need for `Aion` prefix. The prefix is the package name already (`source.Transaction` vs `model.Transaction`)

File: platform/aion/source/model.go [hunk 6]
```diff
@@ -0,0 +1,18 @@
+package source
+
+type AionTransactionRPCResponse struct {
```
Comment (terorie) [inline platform/aion/source/model.go]: How about `type TransactionPage struct` instead?

File: platform/aion/api.go (hunk 3 above)
Comment (terorie) [reply platform/aion/api.go]: Yes, please add an Aion address to the map in the integration test file https://github.com/TrustWallet/blockatlas/blob/master/test/main.go

File: coin/slip44.go (hunk 1 above)
Comment (terorie) [reply coin/slip44.go]: Comments will be added in #50

**Pull request merged**